### DISCUSSION TITLE
Is it possible to use pgxpool to connect to 2 databases at the same time?

### DISCUSSION THREAD

**Discussion by Jimmy99** [jackc/pgx#1290] [Q&A]
In my web app i have 2 databases 1 for user authentication etc. And 1 for the data of the web app. Queries fail when i instantiate 2 connections , but queries succeed when one connection is disabled.  Is there a "correct " way to make 2 connections simultaneously and what is the correct way?

**Comment by jackc** [comment]
You can use two pgxpools at the same time. They won't conflict with each other.
